index.html

```diff
@@ -18,30 +18,38 @@
           email: "marcosc@apple.com",
           company: "Apple Inc.",
           companyURL: "https://apple.com",
+          w3cid: 39125,
+
         },
         {
           name: "Tim Cappalli",
           email: "tim.cappalli@okta.com",
           company: "Okta",
           companyURL: "https://okta.com",
+          w3cid: 111190,
+
         },
         {
           name: "Mohamed Amir Yosef",
           email: "mamir@google.com",
           company: "Google Inc.",
           companyURL: "https://google.com",
+          w3cid: 158857,
+
         },
         {
           name: "Sam Goto",
           email: "goto@google.com",
           company: "Google Inc.",
           companyURL: "https://google.com",
           retiredDate: "2025-05-02",
+          w3cid: 50308,
         },
         // Add additional editors here.
         // https://github.com/w3c/respec/wiki/editors
       ],
       latestVersion: null,
+      canonicalURI: "TR",
```

Comment:
This is not correct... this should be an actual TR URI... I don't think we need this tho. 

```suggestion
```